package-system/AWSNativeSDK/build_AWSNativeSDK_ios.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+src_path=temp/src
+bld_path=temp/build
+
+configure_and_build_static() {
+    build_type=$1
+
+    echo "CMake Configure $build_type Static"
+    CXXFLAGS="-Wno-deprecated-declarations -Wno-shorten-64-to-32 -fPIC" \
+    cmake -S "$src_path" -B "$bld_path/${build_type}_Static" \
+          -DTARGET_ARCH=APPLE \
+          -DCMAKE_SYSTEM_NAME=Darwin \
+          -DCMAKE_OSX_ARCHITECTURES="arm64" \
+          -DCMAKE_OSX_SYSROOT="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk" \
+          -DCMAKE_OSX_DEPLOYMENT_TARGET=13.0 \
+          -DCMAKE_CXX_STANDARD=17 \
+          -DENABLE_TESTING=OFF \
+          -DENABLE_RTTI=ON \
+          -DCUSTOM_MEMORY_MANAGEMENT=ON \
+          -DBUILD_ONLY="access-management;cognito-identity;cognito-idp;core;devicefarm;dynamodb;gamelift;identity-management;kinesis;lambda;mobileanalytics;queues;s3;sns;sqs;sts;transfer" \
+          -DBUILD_SHARED_LIBS=OFF \
+          -DCMAKE_BUILD_TYPE=$build_type \
+          -DCURL_LIBRARY="temp/curl_install/lib/libcurl.a" \
+          -DCURL_INCLUDE_DIR="temp/curl_install/include" \
+          -DCMAKE_INSTALL_LIBDIR="lib/$build_type" || (echo "CMake Configure $build_type Static failed" ; exit 1)
+
+    echo "CMake Build $build_type Static to $bld_path/${build_type}_Static"
+    cmake --build "$bld_path/${build_type}_Static" --config $build_type -j 12 || (echo "CMake Build $build_type Static to $bld_path/${build_type}_Static failed" ; exit 1)
+}
+
+make_configure_and_build_curl() {
+  rm -rf "temp/curl"*
+
+  echo "Downloading Curl 7.65.3"
+  (cd temp && curl -o curl-7.65.3.zip "https://curl.se/download/curl-7.65.3.zip") || exit 1
```

Comment:
Why not clone from Github like what your implementation in the cmake_configure_and_build_curl() function below?

Reply:
source cloned from github repo doesn't support make build directly.